src/invoices/InvoicesList/InvoiceLine/InvoiceLineEditLayer.js

```diff
@@ -78,6 +88,9 @@ class InvoiceLineEditLayer extends Component {
         }],
       };
     const hasLoaded = (!lineId || get(resources, 'invoiceLine.hasLoaded')) && get(resources, 'invoice.hasLoaded');
+    const vendor = get(resources, ['vendor', 'records', 0], {});
```

Comment:
don't you think that you have to wait until vendor is loaded?

Reply:
fixed